content/neuvector-prime/azure/contents.lr

```diff
@@ -169,26 +169,40 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
 
    !["Keyvaluepair config"](./keyvaluepairconfig.png)
 
-## Auto upgrade minor version
 
-By default the Kubernetes Service Extension property "Auto upgrade minor version" is enabled
-during the NeuVector offer deployment. This will allow NeuVector to regularly update
-its CVE database.
+## Install Scanner Updater
 
-Click "Refresh" in the Extension + application screen to view the latest Version.
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry. It is recommended to install scanner updater AFTER successfull initial NeuVector deployment.
```

Comment:
Isn't installing the scanner updater *required* after installing the NeuVector offer?
Also s/successfull/successful/

Reply:
Updated the flow..